atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/KvsSweepQueueReader.java

```diff
@@ -18,39 +18,42 @@
 
 import java.util.Collection;
 import java.util.List;
-import java.util.Optional;
 import java.util.function.Consumer;
 
 import com.google.common.collect.ImmutableList;
-import com.palantir.atlasdb.keyvalue.api.KeyValueService;
-import com.palantir.atlasdb.protos.generated.TableMetadataPersistence;
+import com.palantir.util.Pair;
 
-public class KvsSweepQueueReader implements SweepQueueReader {
-    private final SweepableCellsReader cellsReader;
-    private final SweepableTimestampsReader timestampsReader;
+public final class KvsSweepQueueReader implements SweepQueueReader {
```

Comment:
We could also have a single reader that is passed the relevant `ShardAndStrategy` along with the `consumeNexBatch()` call, since this class isn't stateful. Avoids some extra object allocations, but requires someone a layer up to iterate through the current shard count. I'm not sure which will be cleaner when we get to incrementing the shard count.